plugins/services/src/js/data/MarathonClient.ts

```diff
@@ -51,3 +51,37 @@ export function createGroup(
     })
   );
 }
+
+export function editGroup(
+  id: string,
+  enforceRole: boolean
+): Observable<CreateGroupResponse> {
+  return request(buildMarathonURI("/groups/" + id), {
+    method: "PUT",
+    body: JSON.stringify({ id, enforceRole })
+  }).pipe(
+    map(reqResp => {
+      const { code } = reqResp;
+      if (code > 300) {
+        const response = (reqResp.response || {
+          message: "Unknown"
+        }) as CreateGroupErrorResponse;
+        let message: string;
+        switch (reqResp.code) {
+          case 409:
```

Comment:
I think we can remove 409, since it shouldn't be possible to get a 409 for a `PUT`